src/popup/pages/AccountDetail/index.tsx

```diff
@@ -17,14 +17,25 @@ interface IProps {
 
 @inject('store')
 @observer
-
 class AccountDetail extends Component<WithStyles & IProps, {}> {
+  private static GET_TX_INTERVAL_MS: number = 10000;
+  private getTransactionsInterval?: number = undefined;
```

Comment:
move both vars to store. rule of thumb is to try to keep all biz logic out of components. components should be as "dumb" as possible, get data from store and display.